### PR TITLE
Resume report UI

### PR DESCRIPTION
**Pull request by ajmalbt**
Steps to test:

1. Add a new resource in your scratchorg called ChartJs, and upload the Chart.bundle.min.js

2. Create several applications

3. Upload a few resumes to a few applications

4. Attach the same resumes to the other applications (make sure you don't re-upload from disk the same resume name again, it will actually be a different Salesforce object)

5, Go to the application page, and edit page, search for barchart and add the parentbarchart to the UI

image

### CODE REVIEW

File: force-app/main/default/classes/BarChartController.cls [hunk 1]
```diff
@@ -0,0 +1,101 @@
+public class BarChartController {
+    @AuraEnabled(cacheable=true)
+    public static ReturnWrapper getOpportunities(){
```
Comment (SophieUdud) [inline force-app/main/default/classes/BarChartController.cls:3]: Please rename the method to something more relevant

File: force-app/main/default/lwc/parentbarchart/parentbarchart.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<template>
+    <lightning-card title="Open Opportunities" icon-name="utility:chart">
```
Comment (SophieUdud) [inline force-app/main/default/lwc/parentbarchart/parentbarchart.html:2]: Please rename the title to something more relevant

File: force-app/main/default/lwc/parentbarchart/parentbarchart.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import { LightningElement, wire } from 'lwc';
+import getOpportunities from '@salesforce/apex/BarChartController.getOpportunities'; //@salesforce/apex/Namespace.Classname.apexMethodReference
```
Comment (SophieUdud) [inline force-app/main/default/lwc/parentbarchart/parentbarchart.js:2]: If you update the backend method name, you should also change it here

File: force-app/main/default/lwc/parentbarchart/parentbarchart.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+export default class parentbarchart extends LightningElement {
+    chartConfiguration;
+    @wire(getOpportunities)
+    getOpportunities({ error, data }) {
```
Comment (SophieUdud) [inline force-app/main/default/lwc/parentbarchart/parentbarchart.js:6]: Please also update these 2 lines with the right function/method names

**Review by SophieUdud — COMMENTED** [review]
Ajmal please see the 4 places where I left some comments and address them before merging.
I also added one more commit, where I added the Chart.bundle.min.js as static resource to our project files, so users don't have to add them manually in their org.
Thank you!

**Review by SophieUdud — APPROVED** [review]
Looks good, thank you for addressing the comments!

**Comment by adrienyule** [comment]
Great

**Review by adrienyule — COMMENTED** [review]
Looks good

**Review by adrienyule — APPROVED** [review]
Awesome!

**Pull request merged**